tests/start_test_validator.py

```diff
@@ -21,6 +21,7 @@
         'solana-test-validator',
     ],
     stdout=subprocess.DEVNULL,
+    shell=True,
```

Comment:
Why is this needed? It makes Python execute the command through a shell, so you can do things like put a `~` in the command and have it expand to the value of `$HOME`. But in this case our command doesn’t use any shell features, so I don’t think this does anything.

Reply:
I'm not sure why, but this change makes our CI work, see https://github.com/ChorusOne/solido/pull/541/commits/57bd0fbf348fa299c494c635e4e9967ace212aa4